zello-sdk/src/main/java/com/zello/sdk/Sdk.java

```diff
@@ -67,6 +67,7 @@ class Sdk implements SafeHandlerEvents, ServiceConnection {
 	private static final String _pttActivityClass = "com.zello.sdk.Activity";
 	private static final String _pttPermissionsActivityClass = "com.zello.sdk.PermissionsActivity";
 	private static final String _pttPttButtonsActivityClass = "com.zello.sdk.PttButtonsActivity";
+	private static final String _sdkRequestPermissionsIntentCategory = "com.zello.sdk.SDK_REQUEST_PERMISSION";
```

Comment:
I believe what you intended to do here is call it an action and use it in the place of `android.intent.action.INVISIBLE_ACTIVITY`?
```suggestion
	private static final String _sdkRequestPermissionsIntentAction = "com.zello.sdk.REQUEST_PERMISSIONS";
```